.travis.yml

```diff
@@ -1,14 +1,14 @@
 language: python
 python:
- - "2.6"
```

Comment:
@pypingou Twisted dropped support for 2.6. Since fedmsg uses Twisted the builds will always fail. See https://github.com/fedora-infra/fedmsg/issues/376.